README.rst

```diff
@@ -88,6 +88,12 @@ corotwine work with pytest-twisted::
 That's all.
 
 
+.. |PyPI| image:: https://img.shields.io/pypi/v/pytest-twisted.svg
+    :target: https://pypi.python.org/pypi/pytest-twisted
+
+.. |Pythons| image:: https://img.shields.io/pypi/pyversions/pytest-twisted.svg
+   :alt: supported Python versions
+
```

Comment:
I'm not sure quite what is wrong but these badges don't seem to be loading properly.

https://github.com/hugovk/pytest-twisted/blob/44e308c15510c878b4de2a86e4bd66cbc177550e/README.rst
![image](https://user-images.githubusercontent.com/543719/37289017-08928c12-25df-11e8-9970-bfbcd3021a7d.png)

Maybe it's just a temporary issue and not a source bug?  I don't know.

Reply:
Looks like a temporary problem with shields.io, I saw this on another repo too. And someone just reported this shields.io a few minutes ago:

https://github.com/badges/shields/issues/1568

Would you rather these left out for now, or wait a bit longer?

Reply:
No, leave it.  Maybe update all to the `:target:` form though?  I have little experience with rst and wasn't aware of that.  Definitely seems better though.

Reply:
Done! `:alt:` and `:target:` for all.